platforms/winit/examples/simple.rs

```diff
@@ -43,11 +43,12 @@ fn make_button(id: NodeId, name: &str) -> Node {
     };
```

Comment:
You forgot to remove the now unused `id` argument. It's unfortunate that our CI don't check the examples apparently.